src/bitfield.cpp

```diff
@@ -75,6 +79,30 @@ namespace libtorrent
 		}
 #endif // TORRENT_HAS_SSE
 
+#if TORRENT_HAS_ARM_NEON
+		if (aux::arm_neon_support)
+		{
+			for (int i = 0; i < words; ++i)
+			{
+				uint8x8_t in_val;
```

Comment:
I will rearrange the code, this was to have some resemblance of actual assembly.